src/test/java/com/nerodesk/takes/doc/TkAddFriendTest.java

```diff
@@ -62,7 +62,7 @@ public void addsFriend() throws Exception {
         final String friend = "Bob";
         MatcherAssert.assertThat(
             new RsPrint(new TkAddFriend(doc, friend).act()).print(),
-            Matchers.containsString("document shared")
+            Matchers.containsString("document+shared")
```

Comment:
@darkled yes, there was a change in takes library, and now RsPrint just shows the redirect (HTTP See Other), and the document shared is included only in header as a cookie, and as such it is encoded as in URL